projects/demo/src/modules/components/scrollbar/index.html

```diff
@@ -66,6 +66,13 @@
                 .
             </p>
         </tui-doc-example>
+
+        <tui-doc-example
+            id="auto-hide"
```

Comment:
Let's use ngFor to iterate over examples

Reply:
fixed